osu.Game/Overlays/Chat/DrawableChannel.cs

```diff
@@ -79,6 +82,34 @@ private void load()
             Channel.PendingMessageResolved += pendingMessageResolved;
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            highlightedMessage = Channel.HighlightedMessage.GetBoundCopy();
+            highlightedMessage.BindValueChanged(m =>
```

Comment:
I don't understand why this is a bindable flow rather than an event (ie. `RequestHighlight` similar to what we use in [other](https://github.com/ppy/osu/blob/266b4c7124fe4b578087afaaba46e18ec4ee0423/osu.Game/Overlays/Login/LoginForm.cs#L29) [places](https://github.com/ppy/osu/blob/2a1edcdf73de482ded7335b5ea2fdf2e607ba300/osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs#L57). The value is being nulled as soon as it is consumed anyway, so this either needs further explanation or should be an event flow instead.

Reply:
See previous review conversation: https://github.com/ppy/osu/pull/17137#discussion_r821087954

I found the way this was originally written on events pretty ugly because of the need to potentially handle channel switches at the same time as highlighting the message.